src/demo/demo.js

```diff
@@ -114,50 +80,66 @@ class Demo extends Component{
     isModalOpen: false
   }
 
-  sdkOptions = (clientSdkOptions={})=> ({
-    ...(queryStrings.link_id ?
-      { mobileFlow: true } :
-      {
-        token: this.state.token,
-        useModal,
-        onComplete: (data) => {
-          /*callback for when */ console.log("everything is complete", data)
-        },
-        isModalOpen: this.state.isModalOpen,
-        language,
-        steps,
-        mobileFlow: !!queryStrings.link_id,
-        userDetails: {
-          smsNumber: queryStrings.smsNumber,
-        },
-        onModalRequestClose: () => {
-          this.setState({isModalOpen: false})
-        },
-        ...smsNumberCountryCode,
-        ...clientSdkOptions,
-      }
-    )
-  })
-
   render () {
+    const {
+      containerWidth = '100%',
+      containerHeight = '100%',
+      rootFontSize = '16px',
+      containerFontSize = '16px',
+      tearDown
+    } = this.props.viewOptions || {}
+    // super bad practice, but just setting the sizing on the root node directly
+    rootNode.style = `
+      width: ${containerWidth};
+      height: ${containerHeight};
+      font-size: ${containerFontSize};`
+    document.body.style = `font-size: ${rootFontSize};`
+
+    if (tearDown) return "SDK has been torn down"
+
+    const options = {
+      ...getInitSdkOptions(),
+      ...this.state,
+      onComplete: data => window.parent.postMessage({ type: 'sdk_complete', data }),
```

Comment:
Why snake_case?

Reply:
I don't really know what I was thinking. I wanted these to be enum-y, really, so they should actually be `SDK_COMPLETE` etc... will make the change